plugins/vpc-eni/plugin/commands.go

```diff
@@ -128,7 +128,7 @@ func (plugin *Plugin) Del(args *cniSkel.CmdArgs) error {
 	// If existing network is to be used then ENI is not required.
 	if !netConfig.UseExistingNetwork {
 		// Find the ENI.
-		vpcENI, err = eni.NewENI(netConfig.ENIName, netConfig.ENIMACAddress)
+		vpcENI, err = eni.NewENI(args.IfName, netConfig.ENIMACAddress)
```

Comment:
Let's please confirm with Windows team that this change doesn't impact them. CC @rawahars

Reply:
Thanks @ofiliz and @rawahars!